roles/setup_pgd/vars/edb-ssl.yml

```diff
@@ -0,0 +1,24 @@
+---
+cert_common_name: "PEM"
+cert_country: "US"
+cert_state: "MA"
+cert_city: "Bedford"
+cert_org_unit: "EDB Postgres Enterprise Manager"
+cert_email: "support@enterprisedb.com"
+
+pg_ssl_postgres_conf_params:
+  - {name: "ssl", value: "on"}
+  - {name: "ssl_ca_file", value: "{{ pg_data }}/root.crt"}
+  - {name: "ssl_key_file", value: "{{ pg_data }}/server.key"}
+  - {name: "ssl_cert_file", value: "{{ pg_data }}/server.crt"}
+  - {name: "ssl_crl_file", value: "{{ pg_data }}/root.crl"}
+
+# pg_ssl_ip_addresses:
```

Comment:
I'd prefer commented code to just not be added, unless there is a TODO or FIXME note explaining why it's there.

Reply:
This is intentional. Example of how to use in the edb-ssl.yml